pep-0008.txt

```diff
@@ -241,6 +241,17 @@ Another such case is with ``assert`` statements.
 
 Make sure to indent the continued line appropriately.
 
+
+Error Message String Literals
```

Comment:
I think just saying "Error messages" would be sufficient.  I have to pause to think about what "Error message string literals" are.  Exception strings?  Something else?  Is there something special about string literals for errors?

Reply:
My one concern with that is that readers might think its talking about the wording/formatting of error messages themself, rather than concerned with how string literals for error messages are formatted in the code. But I can see both sides of this.

Reply:
How about just dropping "Literals"?

"Strive to keep error message strings on one line"

vs.

"Strive to keep error messages on one line"

Reply:
PTAL; I've removed "literals".